arch/riscv/kvm/cove.c

```diff
@@ -589,9 +594,9 @@ void noinstr kvm_riscv_cove_vcpu_switchto(struct kvm_vcpu *vcpu, struct kvm_cpu_
 
 	/*
 	 * Bind the vsfile here instead during the new vsfile allocation because
-	 * COVH bind call requires the TVM to be in finalized state.
+	 * COVI bind call requires the TVM to be in finalized state.
 	 */
-	if (tvcpuc->imsic.bind_required) {
+	if (kvm_riscv_cove_capability(KVM_COVE_TSM_CAP_AIA) && tvcpuc->imsic.bind_required) {
```

Comment:
This should be in a separate patch compared to the previous changes. 
All the previous changes enable the Cove capability feature while this actually uses one of those feature.

Reply:
done. The commit #1102393aa3f5b77758f2c36db12855654a23f00e introduces the TSM capabilities feature. The commit #b0c72bb4058fa9dcc0d84af42e227b6c9effe469 modifies the host behavior depending on the TSM capabilities.